backend/src/templates/templates.service.ts

```diff
@@ -138,8 +140,47 @@ export class TemplatesService {
       paramsRequired: templateVersion.paramsRequired,
     }
   }
+  /**
+   * This is so inefficient, I want to die.
+   * Should have just used the query builder...
+   * @param userId
+   * @returns All template metadata which user is editor or issuer of.
+   */
+  async listTemplatesForUser(
+    userId: number,
+  ): Promise<GetTemplateMetaResponseDto[]> {
+    const editorOf = await this.connection.manager.find(Editor, {
+      where: {
+        user: userId,
+      },
+      relations: ['template', 'template.author'],
+    })
 
-  async listTemplatesForUser(): Promise<void> {
-    return
+    const issuerOf = await this.connection.manager.find(Issuer, {
+      where: {
+        user: userId,
+      },
+      relations: ['template', 'template.author'],
+    })
+
+    const templateIds = unionBy(editorOf, issuerOf, (e) => e.template.id).map(
+      (e) => e.template.id,
+    )
+    const templates = await this.connection.manager.find(TemplateVersion, {
+      where: {
+        template: In(templateIds),
```

Comment:
```suggestion
        template: { id: In(templateIds) },
```

Although undocumented, it seems like TypeORM matches the primary key by default so this works, but might be a good idea to make it explicit.